src/server/game/Server/Packets/CalendarPackets.h

```diff
@@ -0,0 +1,576 @@
+            time_t Date = time_t(0);
+            time_t LockDate = time_t(0);
+            uint32 Flags = 0;
+            int32 TextureID = 0;
+            uint8 GetEventType = 0;
+            uint8 EventType = 0;
+            std::string Description;
+            std::string EventName;
+            std::vector<CalendarEventInviteInfo> Invites;
+        };
+
+        class CalendarEventInviteAlert final : public ServerPacket
+        {
+        public:
+            CalendarEventInviteAlert() : ServerPacket(SMSG_CALENDAR_EVENT_INVITE_ALERT, 80) { }
+
+            WorldPacket const* Write() override;
+
+            ObjectGuid OwnerGuid;
+            ObjectGuid EventGuildID;
+            ObjectGuid InvitedByGuid;
+            uint64 InviteID = 0;
+            uint64 EventID = 0;
+            uint32 Flags = 0;
+            time_t Date = time_t(0);
+            int32 TextureID = 0;
+            uint8 Status = 0;
+            uint8 EventType = 0;
+            uint8 ModeratorStatus = 0;
+            std::string EventName;
+        };
+
+        class CalendarEventInvite final : public ClientPacket
+        {
+        public:
+            CalendarEventInvite(WorldPacket&& packet) : ClientPacket(CMSG_CALENDAR_EVENT_INVITE, std::move(packet)) { }
+
+            void Read() override;
+
+            uint64 ModeratorID = 0;
+            bool IsSignUp = false;
+            bool Creating = true;
+            uint64 EventID = 0;
+            std::string Name;
+        };
+
+        class CalendarEventRSVP final : public ClientPacket
+        {
+        public:
+            CalendarEventRSVP(WorldPacket&& packet) : ClientPacket(CMSG_CALENDAR_EVENT_RSVP, std::move(packet)) { }
+
+            void Read() override;
+
+            uint64 InviteID = 0;
+            uint64 EventID = 0;
+            uint8 Status = 0;
+        };
+
+        class CalendarEventInviteStatus final : public ServerPacket
+        {
+        public:
+            CalendarEventInviteStatus() : ServerPacket(SMSG_CALENDAR_EVENT_INVITE_STATUS, 41) { }
+
+            WorldPacket const* Write() override;
+
+            uint32 Flags = 0;
+            uint64 EventID = 0;
+            uint8 Status = 0;
+            bool ClearPending = false;
+            time_t ResponseTime = time_t(0);
+            time_t Date = time_t(0);
+            ObjectGuid InviteGuid;
+        };
+
+        class CalendarEventInviteRemoved final : public ServerPacket
+        {
+        public:
+            CalendarEventInviteRemoved() : ServerPacket(SMSG_CALENDAR_EVENT_INVITE_REMOVED, 29) { }
+
+            WorldPacket const* Write() override;
+
+            ObjectGuid InviteGuid;
+            uint64 EventID = 0;
+            uint32 Flags = 0;
+            bool ClearPending = false;
+        };
+
+        class CalendarEventInviteModeratorStatus final : public ServerPacket
+        {
+        public:
+            CalendarEventInviteModeratorStatus() : ServerPacket(SMSG_CALENDAR_EVENT_INVITE_MODERATOR_STATUS, 29) { }
```

Comment:
16 + 8 + 1 + 1 = 26 (ObjectGuid can have different size, so might be the reason why in sniff its 29)